apps/www2/src/styles/global.css

```
@@ -295,6 +295,21 @@ aside.overflow-y-auto::-webkit-scrollbar {
+/* The committed PDFs get their geometry from `printToPDF`, so nothing ever declared
+   the paper box to a browser printing the page itself. Safari, told nothing, laid the
+   CV out as though a CSS pixel were a point — a uniform 4:3 enlargement that ran 13
+   pages of content to 15 and moved every page break with it.
+
+   The margin is what fixes that, not the size — Safari needs the box spelled out. It
+   is set for paper, roomier than the 12mm the committed PDFs use, because a sheet
+   coming off a printer is read in the hand and has no download button around it.
+   Chrome lets this override what `printToPDF` is passed, so `packages/cv-to-pdf`
+   appends its own `@page` at print time to keep the build pinned to 12mm. */
+@page {
+  size: A4;
```

Stylelint (17.14.1)

[error] Expected "A4" to be "a4" (value-keyword-case)

(value-keyword-case)